deploy/server-setup-guide.md

```diff
@@ -0,0 +1,118 @@
+# Server setup guide
+
+System and network requirements:
+- Ubuntu 18.04 LTS
+- Public static IP address
+
+### Prepeare system:
+#### 1. Upgrade system packages
+```sh
+sudo apt update
+sudo apt upgrade
+```
+#### 2. Install software dependencies
+```sh
+sudo apt install git python3 python3-pip nginx
+```
+#### 3. Install NodeJS v14
+```sh
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
+sudo apt install nodejs
+```
+
+### Set up gunicorn server
+#### 1. Clone the codebase
+```sh
+cd ~
+git clone https://github.com/WebUraXalys/blackout.git
+```
+#### 2. Select the desired branch
+```sh
+git switch <DESIRED BRANCH>
+```
+#### 3. Set up the virtual environment
+```
+python3 -m venv venv
+./venv/bin/activate
```

Comment:
The `./venv/bin/activate` command does not work. Replace this code with `. venv/bin/activate`

Reply:
Fixed.